v4/app/build.gradle

```diff
@@ -18,8 +18,8 @@ android {
         applicationId "exchange.dydx.trading"
         minSdkVersion parent.minSdkVersion
         targetSdkVersion parent.targetSdkVersion
-        versionCode 9
-        versionName "1.0.3"
+        versionCode 10
```

Comment:
FYI, versionCode change doesn't matter here... CI/CD replaces it with build number